po/it.po

```diff
@@ -0,0 +1,196 @@
+   
+#: counter.lua:431  
+msgid "Decrement by 1"  
+msgstr "Decrementa di 1"  
+   
+#: counter.lua:440  
+msgid "Increment by 1"  
+msgstr "Incrementa di 1"  
+   
+#: main.lua:37  
+msgid "No color"  
+msgstr "Nessun colore"  
+   
+#: main.lua:39  
+msgid "Red"  
+msgstr "Rosso"  
+   
+#: main.lua:41  
+msgid "Orange"  
+msgstr "Arancione"  
+   
+#: main.lua:43  
+msgid "Yellow"  
+msgstr "Giallo"  
+   
+#: main.lua:45  
+msgid "Green"  
+msgstr "Verde"  
+   
+#: main.lua:47  
+msgid "Blue"  
+msgstr "Blu"  
+   
+#: main.lua:49  
+msgid "Purple"  
+msgstr "Viola"  
+   
+#: main.lua:146  
+msgid "translator-credits"  
+msgstr "Albano Battistella"  
+   
+#: main.lua:150  
+msgid "Translate this app!"  
+msgstr "Traduci questa app!"  
+   
+#: main.lua:151  
+msgid "Send a tip!"  
+msgstr "Invia una mancia!"  
+   
+#: main.lua:152  
+msgid ""  
+"<p>Various enhancements have been made to allow Tally to be used with a "  
+"screen reader. Keyboard shortcuts for closing counter windows have been "  
+"added.</p>\n"  
+msgstr ""  
+"<p>Sono stati apportati diversi miglioramenti per consentire l'utilizzo di "  
+"Tally con un lettore di schermo. Sono state aggiunte scorciatoie da tastiera "  
+"per chiudere le finestre dei contatori.</p>\n"  
+   
+#: main.lua:157  
+msgid "Main Window"  
+msgstr "Finestra Principale"  
+   
+#: main.lua:160 main.lua:177  
+msgid "Quit Application"  
+msgstr "Esci dall'applicazione"  
+   
+#: main.lua:165 main.lua:200  
+msgid "Keyboard Shortcuts"  
+msgstr "Scorciatoie da Tastiera"  
+   
+#: main.lua:170 main.lua:201  
+msgid "About Tally"  
+msgstr "Informazioni su Tally"  
+   
+#: main.lua:174  
+msgid "Counter Window"  
+msgstr "Finestra Contatore"  
+   
+#: main.lua:182  
+msgid "Close Window"  
+msgstr "Chiudi Finestra"  
+   
+#: main.lua:209  
+msgid "Create a new counter"  
+msgstr "Crea un nuovo contatore"  
+   
+#: main.lua:213  
+msgid "Delete selected counters"  
+msgstr "Elimina i contatori selezionati"  
+   
+#: main.lua:219  
+msgid "Filter counters by name and/or color"  
+msgstr "Filtra i contatori per nome e/o colore"  
+   
+#: main.lua:224  
+msgid "Menu"  
+msgstr "Menu"  
+   
+#: main.lua:228 main.lua:384  
+msgid "Select counters to delete"  
+msgstr "Seleziona i contatori da eliminare"  
+   
+#: main.lua:241  
+msgid "Filter by name…"  
+msgstr "Filtra per nome…"  
+   
+#: main.lua:309  
+msgid "Add to list"  
+msgstr "Aggiungi alla lista"  
+   
+#: main.lua:311  
+msgid "Add this counter to the list"  
+msgstr "Aggiungi questo contatore alla lista"  
+   
+#: main.lua:380  
+msgid "Stop selecting without deleting anything"  
+msgstr "Interrompi la selezione senza eliminare nulla"  
+   
+#: main.c:134  
+#, c-format  
+msgid ""  
+"An unrecoverable error occurred when loading Tally, preventing the program "  
+"from starting.\n"  
+msgstr ""  
+"Si è verificato un errore irrecuperabile durante il caricamento di Tally, "  
+"impedendo l'avvio del programma.\n"  
```

Comment:
Same thing as noted in my other comment. The use of "Tally" can be replaced with "Contatore".